Update language packer to php81 and fix broken functionality (#2)
mautic/language-packer: README.md

This is a command line utility to build installable language packages for Mautic.

### Development Notes

Execute a `composer install` to install the app dependencies.

The `packages` and `translations` directories are gitignored since these are directories our remote resources and installable packages are stored in.

### General Notes and Environment variables

The default application configuration is stored at `.env` and can be overridden by creating a `.env.local` file.

```dotenv
###> symfony/framework-bundle ###
APP_ENV=prod
# change below to your desired secret

> APP_SECRET=c85b90b0d096eb714652f175409489bb

Reviewer: It would be good to know what is the secret for.

### General Notes

> The application configuration is stored at `.env` and should be filled prior to executing this script.  You can copy the `.env.dist` file to start with.

Reviewer 2: ```suggestion
The default application configuration is stored at `.env` and can be overridden by creating a `.env.local` file.
```
